### PR TITLE
User can add exercise to custom workout list.

### PR DESCRIPTION
**Pull request by johnhaup**
# Motivation
User can see an updated Workout Summary in right column of screen.

# Implementation
- [ ] Create function that displays each object in workout object in a div

crunchtime-issue02-complete

### CODE REVIEW

**Comment by johnhaup** [comment]
crunchtime-issue02b-complete

File: main.js [hunk 1]
```diff
@@ -4,35 +4,114 @@ $heading.textContent = "Logo"
...
+$exForms.addEventListener('submit', function(event) {
+  event.preventDefault()
+  var formData = new FormData(event.target)
+  var exName = event.target.dataset.exercise
+  var exSets = formData.get('sets')
+  var exReps = formData.get('reps')
+  var newAdd = {name: exName, sets: exSets, reps: exReps}
+  addToWorkout(newAdd)
+  workout.push(newAdd)
+})
```
Comment (victormejia) [inline main.js]: nice job here

File: main.js [hunk 2]
```diff
@@ -4,35 +4,114 @@ $heading.textContent = "Logo"
...
+  $newInfo.textContent = " Reps: " + newEx.reps + " Sets: " + newEx.sets
+  $newExercise.appendChild($newInfo)
+  console.log($summary)
+  console.log($newExercise)
+  $summary.appendChild($newExercise)
+}
+
+function delExercise (del) {
+//  var toDel = del.dataset.delExercise
+//  console.log(toDel)
```
Comment (victormejia) [inline main.js]: let's delete comments

File: main.js [hunk 3]
```diff
@@ -4,35 +4,114 @@ $heading.textContent = "Logo"
...
+  var $newInfo = document.createElement('div')
+  $newInfo.className = 'panel-body'
+  $newInfo.textContent = " Reps: " + newEx.reps + " Sets: " + newEx.sets
+  $newExercise.appendChild($newInfo)
+  console.log($summary)
+  console.log($newExercise)
+  $summary.appendChild($newExercise)
+}
+
+function delExercise (del) {
```
Comment (victormejia) [inline main.js]: no space between function name and parameter list: `delExercise(del)`

File: main.js [hunk 4]
```diff
@@ -4,35 +4,114 @@ $heading.textContent = "Logo"
...
+  $newHeading.appendChild($newDel)
+  $newExercise.appendChild($newHeading)
+  var $newInfo = document.createElement('div')
+  $newInfo.className = 'panel-body'
+  $newInfo.textContent = " Reps: " + newEx.reps + " Sets: " + newEx.sets
+  $newExercise.appendChild($newInfo)
+  console.log($summary)
+  console.log($newExercise)
+  $summary.appendChild($newExercise)
+}
```
Comment (victormejia) [inline main.js]: remove `console.log`s here

File: main.js [hunk 5]
```diff
@@ -4,35 +4,114 @@ $heading.textContent = "Logo"
...
+  repsLabel.appendChild(reps)
+  var addBtn = document.createElement('button')
+  addBtn.type = 'submit'
+  addBtn.className = 'btn btn-success btn-md'
+  addBtn.textContent = 'Add!'
+  dropsDiv.appendChild(setsLabel)
+  dropsDiv.appendChild(repsLabel)
+  dropsDiv.appendChild(addBtn)
+  form.appendChild(dropsDiv)
+  $exForms.appendChild(form)
```
Comment (victormejia) [inline main.js]: As discussed, let's create helper functions here for things like `reps` and `sets`

File: main.js [hunk 6]
```diff
@@ -4,35 +4,124 @@ $heading.textContent = "Logo"
...
+}
+
+var $summary = document.querySelector('#summary')
+var form = document.querySelector('.exerciseLine')
+var $exForms = document.querySelector('.ex-forms')
+var workout = []
+
+function createSets(exercise) {
+  var setsLabel = document.createElement('label')
+  setsLabel.textContent = "Sets"
```
Comment (victormejia) [inline main.js]: single quote strings

File: main.js [hunk 7]
```diff
@@ -4,35 +4,124 @@ $heading.textContent = "Logo"
...
+
+function createAddBtn(exercise) {
+  var addBtn = document.createElement('button')
+  addBtn.type = 'submit'
+  addBtn.className = 'btn btn-success btn-md'
+  addBtn.textContent = 'Add!'
+  exercise.appendChild(addBtn)
+}
+
+function createForm (muscle) {
```
Comment (victormejia) [inline main.js]: no space here: `createForm(muscle)`

File: main.js [hunk 8]
```diff
@@ -4,35 +4,124 @@ $heading.textContent = "Logo"
...
+  $newExercise.className = 'panel panel-info'
+  var $newHeading = document.createElement('div')
+  $newHeading.className = 'panel-heading'
+  var $newTitle = document.createElement('span')
+  $newTitle.className = 'panel-title'
+  $newTitle.textContent = "Exercise: " + newEx.name
+  $newHeading.appendChild($newTitle)
+  var $newDel = document.createElement('span')
+  $newDel.className = 'glyphicon glyphicon-remove-sign pull-right del-ex'
+  $newDel.dataset.delExercise = newEx.name
```
Comment (victormejia) [inline main.js]: should this just be `name`?

File: main.js [hunk 9]
```diff
@@ -4,35 +4,124 @@ $heading.textContent = "Logo"
...
+}
+
+function delExercise(del) {
+  var toDel = del.dataset.delExercise
+  for (var i = 0; i < workout.length; i++) {
+    if (del === workout[i].name) {
+      delete workout[i]
+    }
+  }
+}
```
Comment (victormejia) [inline main.js]: if this function isn't being used, let's remove it from this PR.

**Pull request merged**